pkgs/by-name/go/goread/package.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub, }:
+
+buildGoModule rec {
+  pname = "goread";
+  version = "1.6.5";
+
+  src = fetchFromGitHub {
+    owner = "TypicalAM";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-SRVXcdgtRpWqvO28CnUcx40nFJnG+Hd94Ezgaj5xK6A=";
+  };
+
+  vendorHash = "sha256-/kxEnw8l9S7WNMcPh1x7xqiQ3L61DSn6DCIvJlyrip0=";
+
+  meta = with lib; {
```

Comment:
is there a specific reason why you'd rather have it without this? 
looking through other packages and merge requests i don't see any specific standard for this used.

Reply:
Because https://github.com/NixOS/nixpkgs/issues/208242. Nixpkgs is not always a good standard, that's why we have to apply treewide changes from time to time.